rclcpp/include/rclcpp/executors/static_single_threaded_executor.hpp

```diff
@@ -190,6 +190,38 @@ class StaticSingleThreadedExecutor : public rclcpp::Executor
     return rclcpp::FutureReturnCode::INTERRUPTED;
   }
 
+  /// Not yet implemented, see https://github.com/ros2/rclcpp/issues/1219 for tracking
+  RCLCPP_PUBLIC
+  void
+  spin_some(std::chrono::nanoseconds max_duration = std::chrono::nanoseconds(0)) override
+  {
+    (void)max_duration;
+    throw rclcpp::exceptions::UnimplementedError(
+            "spin_some is not implemented for StaticSingleThreadedExecutor, use spin or "
+            "spin_until_future_complete");
+  }
+
+  /// Not yet implemented, see https://github.com/ros2/rclcpp/issues/1219 for tracking
+  RCLCPP_PUBLIC
+  void
+  spin_all(std::chrono::nanoseconds) override
```

Comment:
`spin_all()` is not part of the `rclcpp::Executor` class in Foxy.

Reply:
That's what I get for playing fast and loose and not adequately testing this PR. Removed spin_all.